lambdas/service/app.py

```diff
@@ -540,6 +540,10 @@ def validate_repository_search(params, **validators):
     })
 
 
+max_page_size = 1000
```

Comment:
min before max, low before high, left before right etc.

Reply:
@hannes-ucsc would it be appropriate to make the change after a hold-warm tag has been placed on the PR?

Reply:
Not unless I relinquish the PR back to you. I probably won't get around to merging it today, so I'll do just that.

Thanks for asking.